README.md

```diff
@@ -33,7 +33,36 @@ The server process calls `accept()` on the server to accept connect requests fro
 
 So, in order to bootstrap an IPC channel between processes, you create an instance of the `IpcOneShotServer` type, pass the resultant server name into the client process (perhaps via an environment variable or command line flag), and connect to the server in the client. See `spawn_one_shot_server_client()` in `integration_test.rs` for an example of how to do this using a command to spawn the client process and `cross_process_embedded_senders_fork()` in `test.rs` for an example of how to do this using Unix `fork()`[^fork] to create the client process.
 
- ## Implementation overview
+## Testing
+
+To run the tests, issue:
+
+~~~not_rust
+cargo test
+~~~
```

Comment:
I never saw this before. How is this different from
~~~suggestion
```
cargo test
```
~~~
?

Reply:
what if we use `plain` or `console` instead of `not_rust`?

Reply:
Yes, I like `console` as that's actually what is being displayed. Also. I'll switch to the more familiar backtick notation while I'm making the change.